src/app.service.ts

```diff
@@ -1,8 +1,20 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable,  } from '@nestjs/common';
+import { LoggerService } from 'shared/logger/logger.service';
 
 @Injectable()
 export class AppService {
+  
+  constructor(){}
+  private readonly Logger = new LoggerService(AppService.name)
   root(): void {
```

Comment:
How are we setting the context?  this.logger.setContext('context');

Reply:
We can pass the context as constructor param or context as method argument